bundle/tests/scorecard/kuttl/day2operation/01-assert.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: rc.app.stacks/v1beta2
+kind: RuntimeOperation
+metadata:
+  name: example-runtime-operation-0
+spec:
+  podName: day2-operation-rc-0
+  containerName: app
+status:
+  conditions:
+     - status: 'True'
+     - type: Completed
```

Comment:
I believe both `status` and `type` should be part of the same item (remove the `-` infront of `type`). Otherwise, the test may be incorrectly marked as success when in fact it failed (i.e. `Started` type is `True`, but `Completed` is `False`)

```
status:
  conditions:
    - lastUpdateTime: '2021-11-11T17:42:52Z'
      status: 'True'
      type: Started
    - lastUpdateTime: '2021-11-11T17:42:52Z'
      status: 'False'
      type: Completed
```